jackson/src/main/java/com/spotify/hamcrest/jackson/IsJsonBoolean.java

```diff
@@ -58,14 +58,14 @@ protected boolean matchesNode(BooleanNode node, Description mismatchDescription)
     if (booleanMatcher.matches(value)) {
       return true;
     } else {
-      mismatchDescription.appendText("was a boolean node ");
+      mismatchDescription.appendText("was a boolean node with value that ");
       booleanMatcher.describeMismatch(value, mismatchDescription);
       return false;
     }
   }
 
   @Override
   public void describeTo(Description description) {
-    description.appendText("a boolean node ").appendDescriptionOf(booleanMatcher);
+    description.appendText("a boolean node with value that ").appendDescriptionOf(booleanMatcher);
```

Comment:
what would this output if the `jsonBoolean()` overload is called that sets the booleanMatcher to `Matchers.anything()`? 

It looks like it would be `a boolean node with value that ANYTHING` which might be a step in te wrong direction.